services/repository-managers/src/main/java/org/sagebionetworks/repo/manager/message/DataAccessSubmissionStatusMessageBuilderFactory.java

```diff
@@ -0,0 +1,44 @@
+package org.sagebionetworks.repo.manager.message;
+
+import org.sagebionetworks.markdown.MarkdownDao;
+import org.sagebionetworks.repo.model.dataaccess.DataAccessSubmission;
+import org.sagebionetworks.repo.model.dataaccess.DataAccessSubmissionState;
+import org.sagebionetworks.repo.model.dbo.dao.dataaccess.DataAccessSubmissionDAO;
+import org.sagebionetworks.repo.model.message.ChangeType;
+import org.sagebionetworks.util.ValidateArgument;
+import org.springframework.beans.factory.annotation.Autowired;
+
+public class DataAccessSubmissionStatusMessageBuilderFactory implements MessageBuilderFactory {
+
+	public static final String APPROVED_TITLE = "Synapse Notification: Your request had been approved";
+	public static final String APPROVED_TEMPLATE = "A member of the Synapse Access and Compliance Team has reviewed and approved your request.\n"
+			// TODO: verify this link with Jay
+			+"[View your request](https://www.synapse.org/#!Synapse:%1$s)";
```

Comment:
for entity: https://www.synapse.org/#!AccessRequirements:ID={subjectId}
for team: https://www.synapse.org/#!AccessRequirements:teamID={subjectId}